test/Microsoft.AspNetCore.Razor.Tools.Test/MetadataCacheTest.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.Tools
+{
+    public class MetadataCacheTest
+    {
+        [Fact]
+        public void GetMetadata_AddsToCache()
+        {
+            using (var directory = TempDirectory.Create())
+            {
+                // Arrange
+                var metadataCache = new MetadataCache();
+                var assemblyFilePath = LoaderTestResources.Delta.WriteToFile(directory.DirectoryPath, "Delta.dll");
```

Comment:
What's `LoaderTestResources`?

Reply:
It's something @rynowak added to test the AssemblyLoader and ExtensionDependencyChecker.